Ryujinx.HLE/HOS/Services/Nfc/NfcManager/INfc.cs

```diff
@@ -5,6 +5,7 @@ namespace Ryujinx.HLE.HOS.Services.Nfc.NfcManager
     class INfc : IpcService
     {
         private NfcPermissionLevel _permissionLevel;
+        private State _state = State.NonInitialized;
```

Comment:
```suggestion
        private State _state;
```

and `_state = State.NonInitialized;` should be set in the class constructor.